polaris-for-vscode/.vscodeignore

```diff
@@ -1,9 +1,12 @@
+.turbo
+../**
 **/tsconfig.json
 **/*.map
 **/*.ts
 *.vsix
 CONTRIBUTING.md
 docs
+!docs/polaris-icon.png
```

Comment:
this makes sure the icon image can get packaged correctly in the publish process